README.md

```diff
@@ -103,8 +103,7 @@ Bitcoin address: 1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS
 
 Monero address: 48WyAPdjwc6VokeXACxSZCFeKEXBiYPV6GjfvBsfg4CrUJ95LLCQSfpM9pvNKy5GE5H4hNaw99P8RZyzmaU9kb1pD7kzhCB
 
-You will enjoy using FreeTube and hopefully donate using one of the methods. \
-While much appreciated, only donate if you really want to. \
+We hope you enjoy using FreeTube and if you want to support us, donate using one of the methods. \
```

Comment:
This is incorrect grammar and should probably be two sentences. (you cannot combine two conjunctions like this, and these clauses are awkward to combine)